bedrock/firefox/templates/firefox/includes/family-nav.html

```diff
@@ -0,0 +1,153 @@
+{#
+** OFFICIAL™ DOCS **
+
+୧༼ಠ益ಠ༽୨
+
+Preface:
+
+Nav is basically beta right now. Changes will be a-comin' on 3/12. What
+are those changes? We don't know yet! Stay on your toes!
+
+
+1. Add to page:
+  a. Remove old site_header:
+
+    {% block site_header %}{% endblock %}
+
+  b. Add new site_header_unwrapped, including this file:
+
+    {% block site_header_unwrapped %}
+      {% include 'firefox/includes/family-nav.html' %}
+    {% endblock %}
+
+    If you need the dark text version (like for /firefox/partners), set a
+    variable prior to include:
+
+    {% block site_header_unwrapped %}
+      {% set fxfamilynav_class = 'dark' %}
+      {% include 'firefox/includes/family-nav.html' %}
+    {% endblock %}
+
+
+2. Make sure necessary files are in your CSS/JS bundles:
+  a. CSS
+    'css/firefox/family-nav.less'
+
+  b. JS
+    'js/libs/matchMedia.js',
+    'js/libs/matchMedia.addListener.js',
+    'js/libs/jquery.waypoints.min.js',
+    'js/libs/jquery.waypoints-sticky.min.js',
+    'js/firefox/family-nav.js',
+
+
+3. Initialize menu in your page's JS:
+  Quick example:
+
+  var fxNavConfig = {
+    primaryId: 'os', // id of primary nav item to highlight (REQUIRED)
+    subId: 'devices', // id of sub nav item to highlight (OPTIONAL)
+    ctaId: 'id-of-some-button' // id of the CTA to place in sticky nav (OPTIONAL)
+  };
+
+  Mozilla.FxFamilyNav.init(fxNavConfig);
+
+  Available nav/sub nav ID values:
+
+  desktop
+    - index
+    - trust
+    - customize
+    - fast
+  android
+    - index
+  os
+    - index
+    - devices
+    - partners
+    - mwc
+
+  Notes on CTA:
+
+  CTA in sticky nav max dimensions are 160w x 50h. Anything over will be cut off.
+  Conform or feel the wrath of chopped off edges.
+
+  CTA is hidden in sticky nav at tablet (760px) breakpoint - just not enough
+  room (this may change in the future).
+
+  CTA is purposefully hidden for mobile.
+#}
+
+<header id="fxfamilynav-header"{% if (fxfamilynav_class) %} class="{{ fxfamilynav_class }}"{% endif %}>
+  <div class="container">
+    <i id="fxfamilynav-logo">{{ _('Firefox') }}</i>
```

Comment:
Curious on the choice of `<i>` here over a regular heading? No string feelings here, but still feels a little ambiguous when reading http://html5doctor.com/i-b-em-strong-element/